File: app/src/main/java/com/leodeleon/popmovies/di/modules/ActivitiesModule.java
package com.leodeleon.popmovies.di.modules;

import com.leodeleon.popmovies.feature.MainActivity;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module public abstract class ActivitiesModule {

  @ContributesAndroidInjector(modules = FragmentBuildersModule.class)
  abstract MainActivity contributeMainActivity();
}

File: app/src/main/java/com/leodeleon/popmovies/di/modules/ActivitiesModule.kt
package com.leodeleon.popmovies.di.modules

import com.leodeleon.popmovies.feature.MainActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module abstract class ActivitiesModule {

  @ContributesAndroidInjector(modules = arrayOf(FragmentBuildersModule::class))
  internal abstract fun contributeMainActivity(): MainActivity
}
